ssh/tailssh/tailssh.go

```diff
@@ -390,6 +401,20 @@ func (c *conn) ServerConfig(ctx ssh.Context) *gossh.ServerConfig {
 
 			return perms, nil
 		},
+		PasswordCallback: func(cm gossh.ConnMetadata, pword []byte) (*gossh.Permissions, error) {
```

Comment:
But by even having this defined, doesn't that make us advertise it as a valid thing to try next if "none" fails? So then an SSH client can ask for or pop open a password prompt, thinking it might work since we advertised "password" as a next authentication types after the "none" failed?

Reply:
Yes, having this here means that we respond to the `none` auth method with both `publickey` and `password` as acceptable methods.

Reply:
In the interest of minimizing changes, I think we should take this out for now to focus just on achieving parity with 1.80, then we can revisit this question later.

Reply:
Ugh. I miss our NextAuthMethodCallback. :(

Reply:
Oh! We can use `PartialSuccessError` I think!

Reply:
That seems to have done the trick!